src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ApprenticeshipIncentive.cs

```diff
@@ -93,6 +94,36 @@ public void CalculateEarnings(CollectionCalendar collectionCalendar)
             Model.RefreshedLearnerForEarnings = false;
         }
 
+        public void RecalculateEarnings(CollectionCalendar collectionCalendar)
```

Comment:
Do we need this?  Can't the existing CalculateEarnings be called there is duplication here.  Note collectionCalendar isn't used.

Reply:
The requirement is to only recalculate earnings if a payment hasn't already been generated